pkg/cmd/boot/boot.go

```diff
@@ -568,13 +568,17 @@ func (o *BootOptions) verifyRequirements(requirements *config.RequirementsConfig
 
 func (o *BootOptions) verifyClusterConnection() error {
 	client, ns, err := o.KubeClientAndNamespace()
-	if err == nil {
-		_, err = client.CoreV1().Pods(ns).List(metav1.ListOptions{})
```

Comment:
good catch, in the diff I initially did not see the missing error check listing the pods ;-)